edgecast/rtld/real_time_log_delivery_api_client.go

```diff
@@ -34,11 +34,11 @@ const (
 var DefaultSchemes = []string{"http", "https"}
 
 //New creates a new real time log delivery API client
-func New(config edgecast.SDKConfig) (*RealTimeLogDeliveryAPI, error) {
+func New(config edgecast.SDKConfig) (*RtldService, error) {
```

Comment:
I may be mistaken but I think due to naming conventions we should capitalize acronyms. So `RTLDService`